### ISSUE TITLE
Training dataset details about h36m Images

### ISSUE THREAD

**Issue by ouusan** [g-fiche/VQ-HPS#9]
Hello, I am preparing the training dataset following this link you provided https://github.com/pixelite1201/BEDLAM/blob/master/docs/training.md .
 
I would like to ask what exactly is included in your h36m Images. would you mind sharing the contents of your h36m's Images folder (I'm using this just for academic research, and I’m happy to privide my student email if required.)?   

Really appreciate your time and support. Thank you!

**Comment by g-fiche** [comment]
Hello,

The H36M images include the standard training set split (subjects). I suggest downloading the whole dataset; the annotations contain the paths for every image used.

Unfortunately, due to license issues, I cannot share images or annotations for the H36M datasets.

**Comment by ouusan** [comment]
@g-fiche Thank you very much for your quick response ! !

Would it be possible to take a look at the more detailed folder structure of the h36m and mpi-inf-3dhp datasets in your setup?  
Currently, the folder structure shown on the GitHub page only includes something like `images/S1`, `S2`, etc. I also noticed that others have encountered a similar issue when preparing the dataset, as mentioned in this issue: https://github.com/pixelite1201/BEDLAM/issues/14

Would you mind sharing the preprocessing steps you used to prepare these two datasets?

Looking forward to your reply. I would really appreciate your help — thank you again!

**Comment by g-fiche** [comment]
Unfortunately, I do not have access to the folders anymore.

I will add scripts used to extract h36m and mpi-inf-3dhp images in the ```preprocess_data``` folder, I hope this will help.

**Comment by zearo-github** [comment]
I notice that to process a human3.6m dataset using h36m.py, I need h36m_test.npz as input to the h36m_train_extract() function. Could you please publish this file? Or, where can I get it?

**Comment by g-fiche** [comment]
As indicated in the README, we use the annotations from the [BEDLAM](https://github.com/pixelite1201/BEDLAM/tree/master) repository. Training data can be downloaded using ```fetch_real_training_data.sh```, and testing data such as h36m_test.npz with ```fetch_eval_data.sh```.

**Comment by zearo-github** [comment]
I used `fetch_eval_data.sh` to download the data. Is `/data/eval_data_parsed/h36m_valid_protocol1.npz` the same as the `h36m_test.npz` I was referring to? 

![Image](https://github.com/user-attachments/assets/c3bf4296-ab6e-42f0-9013-536fcdc40c45)


However, its keys are not consistent with those of `h36m_train.npz`.

![Image](https://github.com/user-attachments/assets/ec5740aa-31f4-4ad6-b07a-e11e3a3cf279)

**Comment by g-fiche** [comment]
I am not sure, but we only use the training set of H36M. 
We chose to evaluate VQ-HPS on in-the-wild images, and do not evaluate on the test sets of our training data.

It is normal for you to have different keys for the validation set. The SMPL values for H36M are pseudo-ground truth fits that were done for training only. Evaluation is only performed on the 3D joints as other values are not as accurate.
